### PR TITLE
Schedule command

### PR DESCRIPTION
**Pull request by Ttibsi**
I spotted at the end of stream today that someone was looking for a `!schedule` command, so I thought I'd quickly send a PR for it. I wasn't sure if it should include the google calendar link from the about section on twitch as it's a long link and would fill up chat - perhaps just mentioning it in the output message instead? Also useful for users on mobile who can't see the about section of the twitch stream page.

### CODE REVIEW

File: bot/plugins/schedule.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from bot.config import Config
+from bot.data import command
+from bot.data import format_msg
+from bot.message import Message
+
+
+@command('!schedule')
+async def cmd_schedule(config: Config, msg: Message) -> str:
+    return format_msg(
+        msg,
```
Comment (asottile) [inline bot/plugins/schedule.py:12]: bot/plugins/simple.py is more intended for these one-off ones

Comment (Ttibsi) [reply bot/plugins/schedule.py:12]: Thank you, I missed that one. This was a copy of the `Kevin.py` file I changed. I'll add it there and update the PR

File: bot/plugins/simple.py [hunk 2]
```diff
@@ -63,6 +63,12 @@
         'GitHub recently posted a blog post about me: '
         'https://github.com/readme/stories/anthony-sottile',
     ),
+    (
+        '!schedule',
+        'Monday evenings and Saturday at noon (EST)'
+        'Check !twitter and !dicsord for more, or see the google calendar'
+        'link below the stream video',
```
Comment (asottile) [inline bot/plugins/simple.py:70]: this doesn't render properly (missing spaces between words) -- also let's put it in secret commands so the help doesn't get longer

Comment (Ttibsi) [reply bot/plugins/simple.py:70]: Are the secret commands also meant to be alphabetical? It seems like it, but `!question` is above `!pre-commit`, so I could move the former below the p's in this commit too

**Review by asottile — APPROVED** [review]

**Pull request merged**